l2geth/rollup/sync_service.go

```diff
@@ -632,101 +486,171 @@ func (s *SyncService) SetLatestVerifiedIndex(index *uint64) {
 	}
 }
 
-// reorganize will reorganize to directly to the index passed in.
-// The caller must handle the offset relative to the ctc.
-func (s *SyncService) reorganize(index uint64) error {
-	if index == 0 {
-		return nil
-	}
-	err := s.bc.SetHead(index)
-	if err != nil {
-		return fmt.Errorf("Cannot reorganize in syncservice: %w", err)
-	}
```

Comment:
Where did this functionality go? Is it no longer needed?

Reply:
Adding in reorg support will be in a follow up PR, its currently dead code